src/vm/amd64/externalmethodfixupthunk.S

```diff
@@ -11,6 +11,7 @@
 
 NESTED_ENTRY ExternalMethodFixupStub, _TEXT, NoHandler
 
+        set_cfa_register rsp, 8
```

Comment:
Would it be better to make this part of the `PROLOG_WITH_TRANSITION_BLOCK` macro?